apps/molgenis-components/src/components/layout/Molgenis.vue

```diff
@@ -157,8 +157,12 @@ export default {
       );
     },
     menu() {
-      if (this.session && this.session.settings && this.session.settings.menu) {
-        return this.session.settings.menu;
+      if (
+        this.session &&
+        this.session.settingsMap &&
+        this.session.settingsMap.menu
```

Comment:
this.session?.settingsMap?.menu

Reply:
the entire if could be simplified with:
return this.session?.settingsMap?.menu || this.menuItems;